packages/deploy-script-support/src/coreProposalBehavior.js

```diff
@@ -67,59 +94,53 @@ export const makeCoreProposalBehavior = ({
       modules: {
         utils: { runModuleBehaviors },
       },
-    } = allPowers;
-    const [exportedGetManifest, ...manifestArgs] = getManifestCall;
-
-    /** @type {(ref: import('./externalTypes.js').ManifestBundleRef) => Promise<Installation<unknown>>} */
-    const defaultRestoreRef = async ref => {
-      // extract-proposal.js creates these records, and bundleName is
-      // the name under which the bundle was installed into
-      // config.bundles
-      const p =
-        'bundleName' in ref
-          ? E(vatAdminSvc).getBundleIDByName(ref.bundleName)
-          : ref.bundleID;
-      const bundleID = await p;
-      const label = bundleID.slice(0, 8);
-      return E(zoe).installBundleID(bundleID, label);
-    };
-    const restoreRef = overrideRestoreRef || defaultRestoreRef;
+    } = powers;
 
     // Get the on-chain installation containing the manifest and behaviors.
     console.info('evaluateBundleCap', {
       manifestBundleRef,
-      exportedGetManifest,
+      exportedGetManifest: manifestGetterName,
       vatAdminSvc,
     });
     let bcapP;
     if ('bundleName' in manifestBundleRef) {
       bcapP = E(vatAdminSvc).getNamedBundleCap(manifestBundleRef.bundleName);
-    } else {
+    } else if ('bundleID' in manifestBundleRef) {
       bcapP = E(vatAdminSvc).getBundleCap(manifestBundleRef.bundleID);
+    } else {
+      const keys = Reflect.ownKeys(manifestBundleRef).map(key =>
+        typeof key === 'string' ? JSON.stringify(key) : String(key),
+      );
+      const keysStr = `[${keys.join(', ')}]`;
+      throw Error(
+        `bundleRef must have own bundleName or bundleID, missing in ${keysStr}`,
+      );
     }
     const bundleCap = await bcapP;
 
-    const manifestNS = await evaluateBundleCap(bundleCap);
+    const behaviors = await evaluateBundleCap(bundleCap);
 
+    // Get the manifest and its metadata.
     console.error('execute', {
-      exportedGetManifest,
-      behaviors: Object.keys(manifestNS),
+      exportedGetManifest: manifestGetterName,
+      behaviors: Object.keys(behaviors),
     });
```

Comment:
Questions for anyone who can answer: how important are the property names in console output, and why does this call use `console.error` rather than `console.info` or the `log` argument?

Reply:
Debugging cruft that was supposed to be removed a long time ago, but which has been proven useful.  I'd be happy for you to clean it up in whatever way you deem appropriate.

Reply:
Thanks, updated.